modular_zubbers/code/game/machinery/computer/orders/order_computer/cook_order_interdyne.dm

```diff
@@ -0,0 +1,85 @@
+/**
+ * # Syndicate Produce Orders Console
+ *
+ * A produce orders console modified to use a specified department account, and only allow express orders
+ *
+ * This file is based off of order_computer.dm
+ * Any changes made to that file should be copied over with discretion
+ */
+
+/obj/item/circuitboard/computer/order_console/interdyne
+	name = "Interdyne Produce Orders Console"
+	greyscale_colors = CIRCUIT_COLOR_SERVICE
+	build_path = /obj/machinery/computer/order_console/cook/interdyne
+
+/obj/machinery/computer/order_console/cook/interdyne
+	name = "interdyne produce orders console"
+	desc = "An interface for ordering fresh produce and other. The cornerstone of any nutritious meal for the Syndicate crew."
+	circuit = /obj/item/circuitboard/computer/order_console/interdyne
+	order_categories = list(
+		CATEGORY_FRUITS_VEGGIES,
+		CATEGORY_MILK_EGGS,
+		CATEGORY_SAUCES_REAGENTS,
+	)
+	blackbox_key = null
+	forced_express = TRUE
+
+	/// The account to add balance
+	var/credits_account = ACCOUNT_INT
+	/// The resolved bank account
+	var/datum/bank_account/synced_bank_account = null
+
+/obj/machinery/computer/order_console/cook/interdyne/LateInitialize()
+	. = ..()
+	synced_bank_account = SSeconomy.get_dep_account(credits_account == "" ? ACCOUNT_CAR : credits_account)
+
+/obj/machinery/computer/order_console/cook/interdyne/ui_data(mob/user)
+	var/list/data = ..()
+	data["points"] = !synced_bank_account ? 0 : synced_bank_account.account_balance
+	return data
+
+/**
+ * This partically rewrites base behavior for express orders, and to subtract from department account
+ */
```

Comment:
So I am curious, would just giving all of the crew links to the account on their IDs be easier than rewriting all this? I suggest you give it a shot, could make a lot of this code more elegant.

Probably has to be showed into a post_equip on the loadouts.

Reply:
> Perhaps I should default back to the current ID holder's account instead.

We could provide the admiral or whatever the highest rank is, with a card linked to the account so they can sort out payments to the crew if we do this.